SPECS/perl/perl.spec

```diff
@@ -533,9 +542,8 @@ packages like perldoc by perl-Pod-Perldoc and splain by perl-diagnostics.
 Summary:        A module for Perl manipulation of .tar files
 License:        GPL+ or Artistic
 Epoch:          0
```

Comment:
If possible it would be great to get rid of all of the Epochs

Reply:
I just kept them inline with Fedora spec

Reply:
I will not fix them in this upgrade

Reply:
task filed to track this https://microsoft.visualstudio.com/OS/_workitems/edit/48377655